charts/pega/config/deploy/context.xml.tmpl

```diff
@@ -21,20 +21,20 @@
     minEvictableIdleTimeMillis="60000"
     />
 
-  {{ if and .Env.DB_RW_USERNAME .Env.JDBC_RW_URL }}
+  {{ if or .Env.SET_RW .Env.JDBC_RW_URL }}
     <Resource name="jdbc/PegaRULESLongRW"
     auth="Container"
     type="javax.sql.DataSource"
     driverClassName="{{ .Env.JDBC_CLASS }}"
-    url="{{ .Env.JDBC_RW_URL }}"
-    username="{{ .Env.DB_RW_USERNAME }}"
-    password=""
+    url="{{- if .Env.JDBC_RW_URL -}}{{ .Env.JDBC_RW_URL }}{{- else -}}{{ .Env.JDBC_URL }}{{- end -}}?socketTimeout=1200;statement_timeout=1200500;idle_in_transaction_session_timeout=1200500"
```

Comment:
I can 100% guarantee you that this is not valid url syntax on one or more supported platforms, including postgres.  I have some doubts that the properties are all valid across supported platforms in general, but we probably should keep them in the connection properties attr so they at least parse properly.

Reply:
fair enough. Put them back in connection properties. Determined that the last property wins if there are multiple, so appending should be good enough